l3kernel/l3text.dtx

```diff
@@ -145,7 +145,7 @@
 %   \texttt{SpecialCasing.txt}. For \pTeX{}, only the ASCII range is
 %   covered as the engine treats input outside of this range as east Asian.
 %
-%   Language-sensitive conversions are enabled using the \meta{language}
+%   Locale-sensitive conversions are enabled using the \meta{BCP-47}
 %   argument, and follow Unicode Consortium guidelines. Currently, the
 %   languages recognised for special handling are as follows.
```

Comment:
wouldn't that be "locale strings recognized..." or something like that instead of languages?
or  "the BCP47 tags ...."